src/java/io/bazel/rulesscala/test_discovery/DiscoveredTestSuite.scala

```diff
@@ -0,0 +1,45 @@
+package io.bazel.rulesscala.test_discovery
+
+import org.junit.runner.RunWith
+import org.junit.runner.Runner
+import org.junit.runners.Suite
+import org.junit.runners.model.RunnerBuilder
+
+@RunWith(classOf[TextFileSuite])
+class DiscoveredTestSuite
```

Comment:
can you add a comment to the approach here? It looks like this is always registered at the test to run, and then by environmental variable all classes are passed in. Is this right?